packages/client/shared/gqlIds/JiraServerIssueId.ts

```diff
@@ -1,8 +1,8 @@
 const JiraServerIssueId = {
-  join: (providerId: number, integrationRepoId: string, issueId: string) => `${providerId}:${integrationRepoId}:${issueId}`,
+  join: (providerId: number, repoId: string, issueId: string) => `${providerId}:${repoId}:${issueId}`,
   split: (id: string) => {
     const parts = id.split(':')
-    return {providerId: parts[0], integrationRepoId: parts[1], issueId: parts[2]}
+    return {providerId: parseInt(parts[0] ?? '', 10), repoId: parts[1] , issueId: parts[2]}
```

Comment:
+1 `repoId` and `issueId` could be undefined, so when `JiraServerIssueId.split()` is being called, we're adding a default string, e.g. `return api.addComment(comment, issueId ?? '')`.

In my opinion, as `GitHubIssueId` and `JiraIssueId` return strings and `providerId` is a string, we should add a default string for `repoId` and `issueId` in this function too.

Reply:
@nickoferrall Changed to using `!`, assuming the input is valid. As was suggested here: https://github.com/ParabolInc/parabol/pull/6059#discussion_r828400763